couler/core/run_templates.py

```diff
@@ -320,4 +323,24 @@ def run_job(
     # return job name and job uid for reference
     rets = _job_output(step_name, func_name)
     states._steps_outputs[step_name] = rets
+
+    # get image, command from manifest, raise exception if no
+    manifest_dict = yaml.safe_load(manifest)
+    manifest_image = manifest_dict["spec"]["template"]["spec"]["containers"][
```

Comment:
Moving `manifest_dict["spec"]["template"]["spec"]["containers"]` to a separate variable so the format here might look better